language/stdlib/modules/DesignatedDealer.move

```diff
@@ -225,12 +280,34 @@ module DesignatedDealer {
                 dealer.window_inflow == old(dealer.window_inflow) + amount);
         ensures tier_index < len(old(dealer).tiers);
         ensures dealer.window_inflow <= old(dealer).tiers[tier_index];
+
+        include TieredMint<CoinType>;
+    }
+
+    spec schema TieredMint<CoinType> {
+        tc_account: signer;
+        dd_addr: address;
+        amount: u64;
+        tier_index: u64;
+        include Roles::AbortsIfNotTreasuryCompliance{account: tc_account};
+        aborts_if amount == 0 with Errors::INVALID_ARGUMENT;
+        include AbortsIfDealerNotExistAt;
```

Comment:
Betraying my lack of knowledge of the prover here: does `AbortsIfDealerNotExistAt` need to be passed the`dd_addr`?

Reply:
I believe the dd_addr parameter of the including schema is automatically bound to dd_addr parameter of the included schema.  But, in general, if the names are different, an explicit binding will have to be provided.